demo/compose.kinova.yaml

```diff
@@ -1,13 +1,13 @@
 services:
 
   kortex-ros:
-    image: husarion/kortex-joystick:noetic
+    image: husarion/kortex-joystick:nightly
```

Comment:
after building new image it would be nice to change it to some exact version

Reply:
Yes, but this will be another PR I guess